tracer/src/Datadog.Trace.Tools.Runner/LegacySettings.cs

```diff
@@ -39,7 +39,7 @@ internal class LegacySettings : CommandSettings
         public bool SetEnvironmentVariables { get; set; }
 
         [CommandOption("--ci-visibility")]
-        [Description("Run the command in CI Visibility Mode")]
+        [Description("RunAsync the command in CI Visibility Mode")]
```

Comment:
This looks like a bad search and replace.
```suggestion
        [Description("Run the command in CI Visibility Mode")]
```

Reply:
😱   RESHAAAAARPEEEEEEEEER   _shakes fist_